verification/curator-service/ui/src/components/Users.tsx

```diff
@@ -36,14 +69,68 @@ export default class Users extends React.Component<{}, UsersState> {
             });
     }
 
+    updateRoles(event: React.ChangeEvent<{ value: unknown }>, updatedUser: User): Promise<unknown> {
```

Comment:
why do you have to type the returned promise unknown? and are we required to return a promise? can't we just make that method async?

Reply:
Copy/pasting too quickly... Removed the promise.